README.md

```diff
@@ -57,6 +57,7 @@ The following table shows the compatibility of jaeger operator with different co
 
 | Jaeger Operator | Kubernetes           | Strimzi Operator   |
 |-----------------|----------------------|---------------------
+| v1.24           | v1.19+               | v0.23              |
```

Comment:
I think we should fix the versions instead of using "v1.19+". Can you list the versions explicitly?